### ISSUE TITLE
Update FxA Performance Redash Dashboard with Performance Metrics from Payments

### ISSUE THREAD

**Issue by chenba** [mozilla/fxa#3171]
Once https://github.com/mozilla/fxa/issues/3072 is done, we can add performance charts for payments pages to redash.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-498)
┆Priority: Medium
┆Issue Number: FXA-498

**Comment by johngruen** [comment]
@irrationalagent is this something you can own

**Comment by chenba** [comment]
I can own this.  But jbuck said #3072 isn't exactly easy, so it's might be a while before we get to this.

**Comment by chenba** [comment]
Closing this in favor of #3596
